src/mgos_vfs.c

```diff
@@ -1002,3 +1002,25 @@ bool mgos_vfs_gc(const char *path) {
   LOG(LL_INFO, ("%s %p %d", path, fs, ret));
   return ret;
 }
+
+size_t mgos_get_fs_size_path(const char *path) {
```

Comment:
please rename to `mgos_vfs_get_space_total` and make `mgos_get_fs_size` use it

Reply:
done.